_data/simple-icons.json

```diff
@@ -278,6 +278,11 @@
             "hex": "0DBDFF",
             "source": "https://www.adroll.com"
         },
+        {
+            "title": "Advent Of Code",
+            "hex": "FFFF66",
```

Comment:
I'm getting #ffff55 as a color tho... How did you extracted it?
I used the Firefox dev tool integrated color meter. Getting colors from rasters are always a slight gamble, so I would let it pass.

Reply:
I checked the font color of the completed stars in my events. 
<span class="calendar-mark-complete">. I had to double check earlier just to make sure

Reply:
Thank you so much for this quick explanation. I had looked into the favicon, where I got that color in the middle of the star. But having the hex seems more deliberate to me.